models/aliyun_ftasr_api_zh/asr_sdk.py

```diff
@@ -0,0 +1,140 @@
+    # 是否开启智能分轨
+    KEY_AUTO_SPLIT = "auto_split"
+    # 响应参数
+    KEY_TASK = "Task"
+    KEY_TASK_ID = "TaskId"
+    KEY_STATUS_TEXT = "StatusText"
+    KEY_RESULT = "Result"
+    # 状态值
+    STATUS_SUCCESS = "SUCCESS"
+    STATUS_RUNNING = "RUNNING"
+    STATUS_QUEUEING = "QUEUEING"
+    # 创建AcsClient实例
+    client = AcsClient(akId, akSecret, REGION_ID)
+    # 提交录音文件识别请求
+    postRequest = CommonRequest()
+    postRequest.set_domain(DOMAIN)
+    postRequest.set_version(API_VERSION)
+    postRequest.set_product(PRODUCT)
+    postRequest.set_action_name(POST_REQUEST_ACTION)
+    postRequest.set_method('POST')
+    # 新接入请使用4.0版本，已接入（默认2.0）如需维持现状，请注释掉该参数设置。
+    # 设置是否输出词信息，默认为false，开启时需要设置version为4.0。
+    task = {KEY_APP_KEY : appKey, KEY_FILE_LINK : fileLink, KEY_VERSION : "4.0", KEY_ENABLE_WORDS : False}
+    # 开启智能分轨，如果开启智能分轨，task中设置KEY_AUTO_SPLIT为True。
+    # task = {KEY_APP_KEY : appKey, KEY_FILE_LINK : fileLink, KEY_VERSION : "4.0", KEY_ENABLE_WORDS : False, KEY_AUTO_SPLIT : True}
+    task = json.dumps(task)
+    print(task)
+    postRequest.add_body_params(KEY_TASK, task)
+    taskId = ""
+    try :
+        postResponse = client.do_action_with_exception(postRequest)
+        postResponse = json.loads(postResponse)
+        print (postResponse)
+        statusText = postResponse[KEY_STATUS_TEXT]
+        if statusText == STATUS_SUCCESS :
+            print ("录音文件识别请求成功响应！")
+            taskId = postResponse[KEY_TASK_ID]
+        else :
+            print ("录音文件识别请求失败！")
+            return
+    except ServerException as e:
+        print (e)
+    except ClientException as e:
+        print (e)
+    # 创建CommonRequest，设置任务ID。
+    getRequest = CommonRequest()
+    getRequest.set_domain(DOMAIN)
+    getRequest.set_version(API_VERSION)
+    getRequest.set_product(PRODUCT)
+    getRequest.set_action_name(GET_REQUEST_ACTION)
+    getRequest.set_method('GET')
+    getRequest.add_query_param(KEY_TASK_ID, taskId)
+    # 提交录音文件识别结果查询请求
+    # 以轮询的方式进行识别结果的查询，直到服务端返回的状态描述符为"SUCCESS"、"SUCCESS_WITH_NO_VALID_FRAGMENT"，
+    # 或者为错误描述，则结束轮询。
+    statusText = ""
+    while True :
+        try :
+            getResponse = client.do_action_with_exception(getRequest)
+            getResponse = json.loads(getResponse)
+            print ("getResponse", getResponse)
+            statusText = getResponse[KEY_STATUS_TEXT]
+            if statusText == STATUS_RUNNING or statusText == STATUS_QUEUEING :
+                # 继续轮询
+                time.sleep(10)
+            else :
+                # 退出轮询
+                break
+        except ServerException as e:
+            print (e)
+        except ClientException as e:
+            print (e)
+    if statusText == STATUS_SUCCESS :
+        print ("录音文件识别成功！")
+        asr_result = getResponse[KEY_RESULT]
+        return asr_result["Sentences"][0]["Text"]
+    else :
+        print ("录音文件识别失败！")
+        return ""
+
+if __name__ == "__main__":
+    accessKeyId = "您的AccessKey Id"
```

Comment:
Credentials should not be filled in code like these.  Please refer to aliyun_api_zh model, load these credentials from secret files.